content/05.limitations.md

```diff
@@ -1,17 +1,13 @@
 ## Limitations of Study
 
-<!-- We suggest that you divide your limitations section into three steps: (1) identify the study limitations; (2) explain how they impact your study in detail; and (3) propose a direction for future studies and present alternatives.-->
-
-
 This study has some potential limitations.
-The brain tumor samples were collected over decades, RNA samples were prepared using more than one type of library method (stranded or poly-A, **Figure {@fig:S7}A**), and were sequenced at multiple centers.
-While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and possible sequencing center batch effect (**Figure {@fig:S7}C**), we also found a very unbalanced distribution of cancer groups with stranded or poly-A RNA-Seq (**Figure {@fig:S7}A**).
+The brain tumor samples were collected over decades and RNA samples were prepared using more than one type of library method (stranded or poly-A, **Figure {@fig:S7}A**) by multiple sequencing centers.
+While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and possible sequencing center batch effect (**Figure {@fig:S7}C**), we also found a very unbalanced distribution of cancer groups by library preparation type (**Figure {@fig:S7}A**).
 We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences between groups in some cases [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 
```

Comment:
```suggestion
We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences among groups [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 
```